core/src/storage/recovery.py

```diff
@@ -126,11 +126,15 @@ def end_progress() -> None:
     from . import recovery_shares
 
     _require_progress()
-    common.delete(_NAMESPACE, _IN_PROGRESS)
-    common.delete(_NAMESPACE, _DRY_RUN)
-    common.delete(_NAMESPACE, _SLIP39_IDENTIFIER)
-    common.delete(_NAMESPACE, _SLIP39_THRESHOLD)
-    common.delete(_NAMESPACE, _REMAINING)
-    common.delete(_NAMESPACE, _SLIP39_ITERATION_EXPONENT)
-    common.delete(_NAMESPACE, _SLIP39_GROUP_COUNT)
+    for app, key in (
```

Comment:
`for key in (_IN_PROGRESS, _DRY_RUN, ...): common.delete(_NAMESPACE, key)`

Reply:
Done in [6ef97fa](https://github.com/trezor/trezor-firmware/pull/2525/commits/6ef97fabaa52631d48ea7387eaa1b666850c054d)